mongodbforms/fieldgenerator.py

```diff
@@ -115,7 +115,7 @@ def boolean_field(self, value):
         return value.lower() == 'true'
 
     def get_field_label(self, field):
-        if field.verbose_name:
+        if hasattr(field, 'verbose_name') and field.verbose_name:
```

Comment:
`if getattr(field, 'verbose_name', None):`

Reply:
I wouldn't say that's strictly equivalent – the current condition will fail if `field.verbose_name` is an empty string

Reply:
this is, and if you want to nitpick use `getattr(field, 'verbose_name', '')`

Reply:
Nevermind, you're correct 1dc7522 :white_check_mark: